schema/config.model.schema

```diff
@@ -349,6 +349,15 @@
               "right"
             ]
           }
+        },
+        "_iconClass": {
+          "type": "string",
+          "required": false,
+          "default": "icon-list",
+          "title": "Drawer icon class",
+          "inputType": "Text",
```

Comment:
Are we ever likely to use custom _iconClasses? If not is it worth changing this to a select with all available options, a bit easier/nicer to use in authoring tool.

Reply:
We wouldn't be able to customise the lists if we change the icon set for a custom theme.
We'd have to have a list containing all icons or all relevant icons from in the vanilla font
We'd have to maintain separate lists for all places where '_iconClass' is going to be added
We'd have to main those list if any other additional icons are added.
Schema enums cannot yet be reused.